Casks/g/gg.rb

```diff
@@ -0,0 +1,21 @@
+cask "gg" do
+  version "0.27.0"
+  sha256 "56a9973d612c513548bb20ad7ecfb9f7991199139823b0c661f944f55d5873d4"
+
+  url "https://github.com/gulbanana/gg/releases/download/v#{version}/gg_#{version}_universal.dmg"
+  name "gg"
```

Comment:
```suggestion
  name "GG"
```

The app's name seems to be capitalized in the upstream repository.